apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/useMarketDetail.tsx

```diff
@@ -9,13 +9,12 @@ import { ScreenName } from "~/const";
 import useNotifications from "~/logic/notifications";
 import { BaseComposite, StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
 import { MarketNavigatorStackParamList } from "LLM/features/Market/Navigator";
-import MarketDetails from "./MarketDetail";
 
 type NavigationProps = BaseComposite<
   StackNavigatorProps<MarketNavigatorStackParamList, ScreenName.MarketDetail>
 >;
 
-function MarketDetailCont({ navigation, route }: NavigationProps) {
+function useMarketDetail({ navigation, route }: NavigationProps) {
```

Comment:
[COULD] Suffix with ViewModel ? (as this pretty much the role of the hooks, handling the UI logic) 

Same for all new hooks that replace a container

Reply:
Good idea, it will make it easier to understand that a hook is not generic but dedicated to a view